server/sonar-server/src/test/java/org/sonar/server/rule/RuleUpdaterMediumTest.java

```diff
@@ -178,17 +182,20 @@ public void remove_markdown_note() {
     ruleDao.insertOrUpdate(dbSession, ruleDto.getMetadata().setRuleId(ruleDto.getId()));
     dbSession.commit();
 
-    RuleUpdate update = RuleUpdate.createForPluginRule(RULE_KEY).setMarkdownNote(null);
-    underTest.update(update, userSessionRule);
+    RuleUpdate update = RuleUpdate.createForPluginRule(RULE_KEY)
+      .setMarkdownNote(null)
+      .setOrganization(defaultOrganization);
+    underTest.update(dbSession, update, defaultOrganization, userSessionRule);
 
     dbSession.clearCache();
-    RuleDto rule = ruleDao.selectOrFailByKey(dbSession, defaultOrganization.getUuid(), RULE_KEY);
+    RuleDto rule = ruleDao.selectOrFailByKey(dbSession, defaultOrganization, RULE_KEY);
     assertThat(rule.getNoteData()).isNull();
     assertThat(rule.getNoteUserLogin()).isNull();
     assertThat(rule.getNoteCreatedAt()).isNull();
     assertThat(rule.getNoteUpdatedAt()).isNull();
   }
 
+  @Ignore
```

Comment:
Is this test is covered by `UpdateActionTest` then it must be dropped. Ignored tests are not kept.